src/traveler_dashboard.js

```diff
@@ -0,0 +1,30 @@
+import { getSingleTraveler, getAllDestinations } from './travel_tracker_service.js'
+
+function getTravelerInfo(travelerId) {
+  getSingleTraveler().then(travelerInfo => console.log(travelerInfo.travelers[5]))
+    .catch(reason => console.log(reason.message))
+
```

Comment:
Do we need this function?  Is it being called anywhere?  It looks like it is just calling an async function and then console.logging a non dynamically generated travler id?  Doesnt look very useful